include/veriblock/blockchain/blocktree.hpp

```diff
@@ -68,7 +68,9 @@ struct BlockTree {
       return state.addStackFunction("acceptBlockHeader()");
     }
 
-    auto it = block_index_.find(block.previousBlock);
+    // make a copy of hash to let it extend to hash_t size
+    hash_t previousBlockHash(block.previousBlock);
+    auto it = block_index_.find(previousBlockHash);
```

Comment:
why is it needed?

Reply:
Auto conversion does not work here. Probably could be solved with proper Blob constructor but could not find it.